### PR TITLE
4.x post merge type fixes

### PR DESCRIPTION
**Pull request by ADmad**
Fixed typeing errors reported by psalm after `3.next` to `4.x` merge.

### CODE REVIEW

File: src/TestSuite/Constraint/Console/ExitCode.php [hunk 1]
```diff
@@ -57,6 +57,6 @@ public function matches($other): bool
      */
     public function toString(): string
     {
-        return sprintf('matches exit code %d', $this->exitCode);
+        return sprintf('matches exit code %s', $this->exitCode ?? 'null');
```
Comment (dereuromark) [inline src/TestSuite/Constraint/Console/ExitCode.php]: Why is this needed if you  have exist code defined and either null or int?
$this->exitCode should suffice then?

Comment (ADmad) [reply src/TestSuite/Constraint/Console/ExitCode.php]: Nope, cause I want string "null" to be used if exit code is `null`.

Comment (dereuromark) [reply src/TestSuite/Constraint/Console/ExitCode.php]: I see. I didnt see the quotes around it ;) Early for me.
